### DISCUSSION TITLE
Dart package

### DISCUSSION THREAD

**Discussion by zikzak-ai** [oramasearch/orama#205] [Ideas]
Is there an interest or ideal about having a Lyra Dart package? Flutter is a great choice when it comes down to mobile app programming and any app that needs search needs this.

**Comment by micheleriva** [comment]
Hi @zuzu-dev!
I am not really into Dart, but I actually hear a lot of people asking for something like this.
As for now, I feel like I could offer support to those who want to implement such a package!

**Reply by filippomenchini** [reply]
HI! I would like to try to face this challenge.
I don't have much knowledge about full text search engines, it would be interesting to try creating one to learn.
If you have any interesting documents I could read to start understanding this that would be great!

**Comment by zikzak-ai** [comment]
I will try to take a look at source code in the new year. Maybe I can fork-it later. I have one open source package tesseract_ocr. Thanks! ---- On Sun, 18 Dec 2022 13:28:43 +0300  Michele ***@***.***> wrote ---- 
Hi @zuzu-dev! 
I am not really into Dart, but I actually hear a lot of people asking for something like this. 
As for now, I feel like I could offer support to those who want to implement such a package!
